Docs/04_Upgrade-Guides/04_v0.6.X-to-v0.7.X.md

```diff
@@ -98,20 +109,22 @@ class ...
 ## Grid & Group Indices
 
 It is no longer required to access grid and group indices via the `GroupedIndex(|2|3) index` parameter of a kernel.
-Instead, you can access the static properties `Grid.Index(X|Y|Z)` and `Group.Index(X|Y|Z)` from every function in the scope of a kernel.
+Instead, you can access the static properties `Grid.Index(X|Y|Z)` and `Group.Index(X|Y|Z)` from every function in the
+scope of a kernel.
 This simplifies programming of helper methods significantly.
 Furthermore, this also feels natural to experienced Cuda and OpenCL developers.
 
 class ...
 {
-    static void ...(GroupedIndex index)
-    {
-        // Common ILGPU way (still supported)
-        int gridIdx = index.GridIdx;
-        int groupIdx = index.GroupIdx;
+static void ...(GroupedIndex index)
+{
+// Common ILGPU way (still supported)
+int gridIdx = index.GridIdx;
+int groupIdx = index.GroupIdx;
```

Comment:
The white spaces here are important to mimic the correct indentation for c# code.

Reply:
This change should be reverted

Reply:
` ```c#` and ` ``` ` were missing here.